README.md

```diff
@@ -10,6 +10,67 @@ SKGTech is using a [Public Google Calendar](https://www.google.com/calendar/embe
 
 If you want to contribute to the development of this site you first need to setup your environment based on the project's requirements.
 
+### Use Docker for development
+
+If you are tired of installing npm, gem and other stuff there is a [Docker](http://docker.com) [image](https://hub.docker.com/r/skgtech/website/) for you.
+
+See more at [github.com/skgtech/skgtech.io-docker](https://github.com/skgtech/skgtech.io-docker/).
+
+Here is an example of steps of using this Docker image.
+
+Pull the latest docker image
+```
+git pull skgtech/website
```

Comment:
The info are to use Docker to build the SKGTech.io website. Not to build any docker image etc. 
Did not understand well?